code/modules/mob/living/simple_animal/simple_animal.dm

```diff
@@ -411,6 +411,7 @@
 		icon_state = icon_dead
 		stat = DEAD
 		density = 0
+		lying = 1
```

Comment:
This will probably have some issues with player controlled simple animals. Hopefully nothing major.